man/rgblink.5

```diff
@@ -1,53 +1,37 @@
 .\" SPDX-License-Identifier: MIT
 .\"
-.Dd March 28, 2021
+.Dd December 22, 2023
 .Dt RGBLINK 5
 .Os
 .Sh NAME
 .Nm rgblink
 .Nd linker script file format
 .Sh DESCRIPTION
-The linker script is an external file that allows the user to specify the order of sections at link time and in a centralized manner.
+The linker script is a self-contained file that allows specifying attributes for sections at link time, and in a centralized manner.
+There can only be one linker script per invocation of
+.Nm ,
+but it can be split into several files
+.Pq using the Ic INCLUDE No directive .
+.Ss Basic syntax
+A linker script follows a very line-oriented format: each line can contain up to one directive or section name, and up to one comment.
+Whitespace (space and tab characters) is used to separate elements, but is otherwise ignored.
 .Pp
-A linker script consists of a series of bank declarations, each optionally followed by a list of section names (in double quotes) or directives.
-All reserved keywords (bank types and directive names) are case-insensitive; all section names are case-sensitive.
-.Pp
-Any line can contain a comment starting with
+Comments begin with a semicolon
 .Ql \&;
-that ends at the end of the line.
-.Bd -literal -offset indent
-; This line is a comment
-ROMX $F            ; start a bank
-  "Some functions" ; a section name
-  ALIGN 8          ; a directive
-  "Some array"
-
-WRAMX 2            ; start another bank
-  org $d123        ; another directive
-  "Some variables"
-.Ed
-.Pp
-Numbers can be in decimal or hexadecimal format
-.Pq the prefix is Ql $ .
-It is an error if any section name or directive is found before setting a bank.
+character, until the end of the line; they are simply ignored.
```

Comment:
I also use semicolons a lot, but in one paragraph like this they're probably not necessary (and rgbasm(5)'s similar section uses short sentences).
```suggestion
character, until the end of the line. They are simply ignored.
```